doozerlib/distgit.py

```diff
@@ -1612,6 +1612,50 @@ def _mapped_image_for_assembly_build(self, parent_images, i):
         unique_pullspec += f':{parent_build_nvr["version"]}-{parent_build_nvr["release"]}'
         return unique_pullspec
 
+    def _mapped_image_from_stream(self, image, original_parent, dfp):
```

Comment:
could there be some `logger.debug` statements in here, and probably even a `logger.info` for when the upstream does override our config? in my tests it wasn't easy to see why the upstream override isn't happening as expected. actually i don't think it's even getting here

Reply:
Added in https://github.com/openshift/doozer/pull/646/commits/98d9760e0b1b4eb25f02588e22a6a1e6b4fd8b3b